pageStyles/login.module.scss

```diff
@@ -0,0 +1,21 @@
+﻿$font-size: 60px;
+$font-family: sans-serif;
+$font-color: orange;
+
+.h1 {
+  font-size: $font-size;
+  font-family: $font-family;
+  color: $font-color;
+  text-align: center;
+}
+
+.h2{
```

Comment:
Are there more meaningful names the SCSS classes could have? Something like `page-header`, `sub-header`, `content-text` etc.?